test-requirements.txt

```diff
@@ -0,0 +1 @@
+bandit==1.4.0
```

Comment:
Do we want to move this into the `requirements-dev.txt` file or just keep it outside as is here?